assignments/function-conversion.js

```diff
@@ -5,26 +5,51 @@
 // };
 // myFunction();
 
+let myFunction = () => {
+console.log("Function was invoked!");
+};
+myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+ let anotherFunction = (param) => {
+   return param;
+ };
+ anotherFunction("Example");
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+let add = (param1, param2) => {
+   return param1 + param2;
+ };
+ add(1,2);
```

Comment:
Remember that you can use implicit returns with arrow functions. So instead of writing it as done above, you could also do something like this:
```js
let add = (param1, param2) => param1 + param2;
```